app/controllers/main/routes.py

```diff
@@ -85,33 +85,33 @@ def events(selectedTerm, activeTab, programID):
     term: Term = Term.get_by_id(currentTerm)
 
     currentEventRsvpAmount = getEventRsvpCountsForTerm(term)
-    studentLedEvents = getStudentLedEvents(term)
-    countUpcomingStudentLedEvents = getUpcomingStudentLedCount(term, currentTime)
+    volunteerOpportunities = getVolunteerOpportunities(term)
+    countUpcomingVolunteerOpportunities = getUpcomingVolunteerOpportunitiesCount(term, currentTime)
     trainingEvents = getTrainingEvents(term, g.current_user)
     engagementEvents = getEngagementEvents(term)
     bonnerEvents = getBonnerEvents(term)
-    otherEvents = getOtherEvents(term)
+    celtsLabor = getCeltsLabor(term)
 
     managersProgramDict = getManagerProgramDict(g.current_user)
 
     # Fetch toggle state from session    
     toggleState = request.args.get('toggleState', 'unchecked')
 
-    # compile all student led events into one list
+    # compile all volunteer opportunitiesevents into one list
     studentEvents = []
-    for studentEvent in studentLedEvents.values():
+    for studentEvent in volunteerOpportunities.values():
         studentEvents += studentEvent # add all contents of studentEvent to the studentEvents list
 
     # Get the count of all term events for each category to display in the event list page.
-    studentLedEventsCount: int = len(studentEvents)
+    volunteerOpportunitiesCount: int = len(studentEvents)
     trainingEventsCount: int = len(trainingEvents)
     engagementEventsCount: int = len(engagementEvents)
     bonnerEventsCount: int = len(bonnerEvents)
-    otherEventsCount: int = len(otherEvents)
+    celtsLaborCount: int = len(celtsLabor)
 
     # gets only upcoming events to display in indicators
     if (toggleState == 'unchecked'):
-        studentLedEventsCount: int = sum(list(countUpcomingStudentLedEvents.values()))
+        volunteerOpportunitiesCount: int = sum(list(countUpcomingVolunteerOpportunities.values()))
```

Comment:
Using type annotations is fine, but it doesn't make sense to re-annotate the type here. We've already declared what we want the type for `volunteerOpportunitiesCount` to be, so we don't need to say it again here. The interpreter will ensure that the type stays the same (that's the point of using the type annotations).